diagnostic.md

```diff
@@ -48,11 +53,16 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+    <!-- I'm unclear on how to answer but i think '/movies/:movie_id/123  -->
     ```
```

Comment:
Remember, the route object has access to route params. Thus, in your route.js file, the model method can be passed a  parameter of `params` where `params.movie_id` can be accessed.